code/src/containers/tables/products-table/components/products-table-body/ProductsTableBody.tsx

```diff
@@ -0,0 +1,66 @@
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+
+import { productStatuToTranslationKeyMap } from "@/containers/tables/products-table/ProductsTable.constants";
+import { ProductsTableBodyProps } from "@/containers/tables/products-table/ProductsTable.types";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppIconButton from "@/components/app-icon-button/AppIconButton";
+import { AppTableCell } from "@/components/app-table/components";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import formatDate from "@/utils/format-date/formatDate";
+import formatPrice from "@/utils/format-price/formatPrice";
+import getCategoryFromTags from "@/utils/get-category-from-tags/getCategoryFromTags";
+
+import "@/containers/tables/products-table/components/products-table-body/ProductsTableBody.scss";
+
+const ProductsTableBody = ({ product }: ProductsTableBodyProps) => {
+  const { name, imageLink, price, quantity, status, tags, createdAt } = product;
+
+  const categoryName = getCategoryFromTags(tags) || "-";
```

Comment:
Maybe would be better to move this condition inside the function to not duplicate this check for each time?

Reply:
I think it makes function more reusable. Despite in these cases we set `-` if there are no category, in other cases we could not need to set `-`